src/main/resources/com/cloudbees/simplediskusage/QuickDiskUsagePlugin/refresh-disk-usage.js

```diff
@@ -0,0 +1,13 @@
+function refreshDiskUsage(a) {
+
+    var xhr = new XMLHttpRequest();
```

Comment:
It feels a bit weird to me that we're switching to older APIs.

Regarding the errors that you're getting with `fetch` my guess is that the execution order is different in old and new version. `callback` sounds like something that will be executed after the main action, which is following the href in case of anchor tag. While in `onclick` that JavaScript would be executed before following the href.

Reply:
Yaroslav's explanation seems spot on. I just checked and couldn't reproduce the issue.

I cannot say I'm thrilled to go back to an older API :/

Unfortunately since I couldn't reproduce on 2 different browsers, I'm having a hard time giving additional advice. What's for sure is that, as Basil said, we do not want to catch and drop every error, it's bad and might hide a real issue.
What we could consider, however, is to handle this specific error if we understand it.

Do you have additional details about the exception? When the exception is thrown, is the POST request still sent?

Reply:
@PierreBtz

I tested the issue in Chrome and was unable to reproduce it. It appears this issue is specific to Safari, and I apologize for not testing in other browsers earlier. The error details were quite limited, but it's clear that Safari was causing the problem, even though the server handled the page refresh correctly, and the POST requests were sent without issue.

In Chrome, the fetch API works as expected.